internal/store/fleet.go

```diff
@@ -108,6 +111,20 @@ func (s *FleetStore) List(ctx context.Context, orgId uuid.UUID, listParams ListP
 	return &apiFleetList, result.Error
 }
 
+func (s *FleetStore) ListOwnedBy(ctx context.Context, orgId uuid.UUID, owner string) (*api.FleetList, error) {
+	var fleets model.FleetList
+	log := log.WithReqIDFromCtx(ctx, s.log)
+
+	condition := model.Fleet{Resource: model.Resource{OrgID: orgId, Owner: util.StrToPtr(owner)}}
+	result := s.db.Model(condition).Where(condition).Find(&fleets)
+	fleetList := fleets.ToApiResource(nil, nil)
+	if result.Error != nil {
+		return &fleetList, result.Error
+	}
+	log.Printf("db.Find(): %d rows affected, error is %v", result.RowsAffected, result.Error)
+	return &fleetList, nil
+}
```

Comment:
This is just a special case of List().  See my changes here in internal/store/common.go:

https://github.com/flightctl/flightctl/pull/90/files#diff-1c6e9eca029b27fdaae737726193fe389fab0692a101f2e467fc6a4fc59871a3

Reply:
Ill wait for your PR to be merged